sdk/core/azure-core/src/main/java/com/azure/core/implementation/RestProxy.java

```diff
@@ -505,7 +507,7 @@ protected Object handleHttpResponse(final HttpRequest httpRequest, Mono<HttpDeco
 
     protected Object handleResumeOperation(HttpRequest httpRequest, OperationDescription operationDescription, SwaggerMethodParser methodParser, Type returnType, Context context)
         throws Exception {
-        throw new Exception("The resume operation is not available in the base RestProxy class.");
+        throw logger.logExceptionAsError(new RuntimeException("The resume operation is not available in the base RestProxy class."));
```

Comment:
Need double confirm that change Exception to RuntimeException. If can't, will suppress this error